tensorbay/client/dataset.py

```diff
@@ -464,16 +464,26 @@ def commit(self, message: str, *, tag: Optional[str] = None) -> None:
         self._status.check_authority_for_draft()
         self._status.checkout(commit_id=self._commit(message, tag))
 
-    def update_notes(self, *, is_continuous: bool) -> None:
+    def update_notes(
+        self,
+        *,
+        is_continuous: Optional[bool] = None,
+        bin_point_cloud_fields: Optional[Iterable[str]] = None,
+    ) -> None:
         """Update the notes.
 
         Arguments:
             is_continuous: Whether the data is continuous.
+            bin_point_cloud_fields: The field names of the bin point cloud files in the dataset.
 
         """
         self._status.check_authority_for_draft()
 
-        patch_data: Dict[str, Any] = {"isContinuous": is_continuous}
+        patch_data: Dict[str, Any] = {}
+        if is_continuous is not None:
+            patch_data["isContinuous"] = is_continuous
+        if bin_point_cloud_fields:
+            patch_data["binPointCloudFields"] = bin_point_cloud_fields
```

Comment:
`bin_point_cloud_fields` is `Iterable` according to the type hint.
an `iterator` is also `Iterable`, but `iterator` is not JSON serializable.
That means the request will failed.